### PR TITLE
Feat: Fix gitignore file,added contributing.md and license file

### LINKED ISSUE #27: Draft a Contributing Guide Template
Create a CONTRIBUTING.md file with guidelines on how to contribute to the project.

### PR DESCRIPTION
**Pull request by Rajkumar-justcoder**
Files added :
- LICENSE 
- CONTRIBUTING.md

Fixed .gitignore file which contains remark of old conflict 

Do let me know it any changes are required to do 

Edit: Fix #27

### CODE REVIEW

**Comment by amos-kibet** [comment]
@Rajkumar-justcoder, request for review so that @sankaire or myself can review your code.

**Comment by Rajkumar-justcoder** [comment]
oh okay

**Comment by Rajkumar-justcoder** [comment]
i cant request you or sankaire for review request

image
No options for that

**Comment by amos-kibet** [comment]
Confirm that clicking on Reviewers text does nothing.

**Comment by amos-kibet** [comment]
and I will look into it

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+## Setup Installation 
+
+Rename the file named `.env.example` to `.env` and update the variable values with valid ones
+
+Install the required dependencies with
+
+```sh
+npm install
+```
```
Comment (amos-kibet) [inline CONTRIBUTING.md:70]: You can also add options for using yarn

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+Start the app with
+
+```sh
+npm start
+```
```
Comment (amos-kibet) [inline CONTRIBUTING.md:76]: Same here

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+
+You can also start it in watch mode with
+
+```sh
+npm run start:dev
+```
+
```
Comment (amos-kibet) [inline CONTRIBUTING.md:83]: Here as well

**Review by kerenketepela — COMMENTED** [review]
can we add the Request for a review step

File: LICENSE [hunk 4]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2022 amos-kibet
+
```
Comment (amos-kibet) [inline LICENSE:4]: I would prefer name of the app instead of my name on the copyright line 3 above

Comment (amos-kibet) [reply LICENSE:4]: mysic api

**Review by amos-kibet — CHANGES_REQUESTED** [review]
Kindly check on those comments, and we should be good

File: CONTRIBUTING.md (hunk 2 above)
Comment (Rajkumar-justcoder) [reply CONTRIBUTING.md:76]: Ah , i thought that any thing would be work as all are same and lock files of that would be already added to gitignore file 

Will change it soon

File: LICENSE (hunk 4 above)
Comment (Rajkumar-justcoder) [reply LICENSE:4]: Okay noted

**Comment by Rajkumar-justcoder** [comment]
would u like to add lock files to .gitignore related to package-lock.json and pnpm lock file ?

**Comment by amos-kibet** [comment]
Sure, those can be ignored

**Comment by Rajkumar-justcoder** [comment]
> Confirm that clicking on Reviewers text does nothing.

yup it does not open any pop up or dropdown

**Comment by amos-kibet** [comment]
> > Confirm that clicking on Reviewers text does nothing.
> 
> yup it does not open any pop up or dropdown

No worries. I can still review your code. It must be some repo settings.

**Comment by Rajkumar-justcoder** [comment]
> 

I am also thinking that only 

btw i have changed requested changes

**Review by amos-kibet — APPROVED** [review]

**Comment by amos-kibet** [comment]
Looks good. Let  me merge.

**Comment by amos-kibet** [comment]
Good work @Rajkumar-justcoder

**Comment by amos-kibet** [comment]
You can take up another issue.

**Comment by Rajkumar-justcoder** [comment]
> Good work @Rajkumar-justcoder

Thanks You :people_hugging:

**Comment by Rajkumar-justcoder** [comment]
> You can take up another issue.

I have checked some issue but i am not familiar with that issues or that tech

**Comment by Rajkumar-justcoder** [comment]
I Guess i can work on .env file issue if i can go thorough code and see it

**Comment by amos-kibet** [comment]
> > You can take up another issue.
> 
> I have checked some issue but i am not familiar with that issues or that tech
No worries. In case you need help or clarification, just ping @sankaire or myself

**Comment by Rajkumar-justcoder** [comment]
> > > You can take up another issue.
> > 
> > 
> > I have checked some issue but i am not familiar with that issues or that tech
> > No worries. In case you need help or clarification, just ping @sankaire or myself

Okay sure

**Comment by amos-kibet** [comment]
> I Guess i can work on .env file issue if i can go thorough code and see it

Try that. 
Hint: all env variables in the code are referrenced by process.env. Example: with **process.env.PORT**,  PORT is an env variable.

**Comment by Rajkumar-justcoder** [comment]
> > I Guess i can work on .env file issue if i can go thorough code and see it
> 
> Try that. Hint: all env variables in the code are referrenced by process.env. Example: with **process.env.PORT**, PORT is an env variable.

yup i was going to  search by  that only :+1:

**Comment by amos-kibet** [comment]
> 

Cool

**Pull request merged**